pkg/apis/direct.csi.min.io/v1alpha1/types.go

```diff
@@ -61,7 +62,7 @@ type DirectCSIDrive struct {
 	// +optional
 	DriveStatus DriveStatus `json:"driveStatus,omitempty"`
 	// +optional
-	Topology *csi.Topology `json:"topology,omitempty"`
+	Topology map[string]string `json:"topology,omitempty"`
```

Comment:
Cannot use csi.Topology because DeepCopy is not implemented for it. Defaulting this to map[string]string